docroot/themes/custom/mass_theme/templates/paragraphs/paragraph--csv-table.html.twig

```diff
@@ -42,23 +42,31 @@
   '#format':  paragraph.field_csvtable_caption.format ?: 'basic_html',
 } %}
 
-{% include "@atoms/09-media/figure.twig" with {
-  figure: {
-    'title': {
-      'text': paragraph.field_csvtable_title.value,
-      'visible': true,
-      'level': 3
-    },
+{% set figure = {
     'align': '',
     'wrap': FALSE,
     'size': paragraph.field_csvtable_display_size.value,
     'class': 'ma__csvtable',
     'ariaLive': 'polite',
-    'caption': caption,
+    'caption': caption|render|striptags|trim ? caption : '',
```

Comment:
what does `striptags` do? Just making sure that it doesn't strip away html tags added in rich txt

Reply:
@clairesunstudio  It is not removing anything, it is just checking it is not empty. If it is not empty, then it returns the caption as is.